### DISCUSSION TITLE
Pull sway config to Arch

### DISCUSSION THREAD

**Discussion by jjeziorny** [Manjaro-Sway/manjaro-sway#881] [General]
Hi,
Great work here, I've installed Manjaro Sway edition and was very impressed by it.

However, I do like the idea that Arch has newest kernel version so would like to use it with the Sway configuration you have done here.

So I was wondering if there is an easy way to pull the required packages and config files from your distribution into a vanilla arch system?

**Comment by boredland** [comment]
You can basically just switch to `unstable`, which is ~always having the latest kernels that arch has: 

`sudo pacman-mirrors --api --set-branch unstable`

I am always on `unstable` for years and already am using `6.12.1`

**Reply by jjeziorny** [reply]
Thanks, actually noticed I can switch to different kernel version as well. so switched to 6.12

I do have one other issue that happened to me during Manjaro installation, but guess I'll open a new thread for that.
When trying to setup encrypted LVM, system is very slow to boot. It shows luks prompt on cli (not gui) and after typing it, takes over a minute to actually start booting.
